README.md

```diff
@@ -48,6 +49,16 @@ go run k8s/examples/simple/env.go
 ```
 For more features follow [tutorial](./k8s/TUTORIAL.md)
 
+#### With TOML config
+It should be noted that using env vars for configuring CL nodes in k8s is deprecated. TOML config should be used instead:
+```toml
+[ChainlinkImage]
+image="public.ecr.aws/chainlink/chainlink"
```

Comment:
can it be considered as secret?

Reply:
could you expand on that? why would we consider image name a secret? in this very case this is a public ecr repo, so not much of a secret.

In GHA in all jobs that use images stored on our private ECR we should mask make the image (and in core that's what we do in all jobs)

Reply:
There can be scenarios where in on-demand jobs, one might want to run a test against specific image input. For example I sometime run test with specific ECR image which is not stored in QA ECR but in production ECR. In that case masking the image becomes necessary.

Reply:
but if our AWS account or region is secret GH will mask them anyway in logs, right?